genie-docs/src/docs/asciidoc/_properties.adoc

```diff
@@ -13,6 +13,23 @@ The Spring properties described here are ones that we have overridden from Sprin
 |===
 |Property |Description| Default Value
 
+
+|genie.agent.filter.enabled
+|If set to `true`, enables the built-in agent filter service. The filter behavior is controlled by other active `genie.agent.filter.*` properties.
+|
+
+|genie.agent.filter.version.minimum
+|(Dynamic) The minimum version an agent needs to be (e.g., `1.2.3`) in order to communicate with this server. The filter needs to be enabled for this to take effect.
```

Comment:
Rather than `(Dynamic)` why don't we add another column to the table that just says whether it's true or false?

Reply:
Added a commit that moves this to a separate column